### PR TITLE
Ienkovich/sort values

### PR DESCRIPTION
**Pull request by ienkovich**

### CODE REVIEW

File: modin/experimental/backends/omnisci/query_compiler.py [hunk 1]
```diff
@@ -405,6 +405,46 @@ def insert(self, loc, column, value):
...
+        ascending : bool
+            Sort in ascending order (True) or descending order (False)
+
+        Returns
+        -------
+        PandasQueryCompiler
+            A new query compiler that contains result of the sort
+        """
+        raise NotImplementedError(
+            "column orting is not yet suported in DFAlgQueryCompiler"
```
Comment (Garra1980) [inline modin/experimental/backends/omnisci/query_compiler.py:445]: typo

File: modin/pandas/test/test_dataframe.py [hunk 2]
```diff
@@ -5736,6 +5736,32 @@ def test_sort_values(self, request, data, axis, ascending, na_position):
             )
             df_equals(modin_df_cp, pandas_df_cp)
 
+    def test_sort_values_with_duplicates(self):
```
Comment (Garra1980) [inline modin/pandas/test/test_dataframe.py:5739]: Should go to master?

Comment (Garra1980) [reply modin/pandas/test/test_dataframe.py:5739]: Or it was just cherry picked from there?

Comment (ienkovich) [reply modin/pandas/test/test_dataframe.py:5739]: it is picked commit from master

File: modin/experimental/backends/omnisci/query_compiler.py (hunk 1 above)
Comment (ienkovich) [reply modin/experimental/backends/omnisci/query_compiler.py:445]: fixed

**Pull request merged**